tests/contrib/executors/test_kubernetes_executor.py

```diff
@@ -541,6 +542,16 @@ def test_worker_container_dags(self):
         self.assertEqual(0, len(dag_volume_mount))
         self.assertEqual(0, len(init_containers))
 
+    def test_mount_current_environment(self):
```

Comment:
you're not testing the feature used in this section as far as I can tell `[kubernetes_environment_variables]`